tensorboard/tools/do_not_submit_test.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/sh
+# Copyright 2018 The TensorFlow Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+
+# Check that the capitalized strings "do not submit" and "do_not_submit"
+# do not appear in any code files.
+
+! grep -r -e 'DO NOT'' ''SUBMIT' -e 'DO_NOT''_''SUBMIT' \
+    --include='*.*' \
+    --exclude='*.{pyc,json,png,wav,pbtxt,md,in,rst,cfg,ipynb}' \
+    tensorboard
```

Comment:
Okay, sure. Changing it to `grep -r .` has the somewhat unfortunate
property that the script will silently sample fewer files if run from
the non-root directory (whereas with `tensorboard/` it would likely
fail), so I’ll add the appropriate logic to cd to the root directory.

Reply:
That wouldn’t work in google3 (different directory structure)—which may
not be a hard requirement, but is nice to have when the cost is this
low.